frontend/src/components/dashboard/DashboardTransactions.tsx

```diff
@@ -4,7 +4,7 @@ import React from "react";
 import { Row, Col } from "react-bootstrap";
 
 import * as T from "../../libraries/explorer-wamp/transactions";
-import { DatabaseConsumer } from "../../context/DatabaseProvider";
+import { ListConsumer } from "../../context/ListProvider";
```

Comment:
List of what? Please-please-please, give descriptive names. `DashboardDataProvider` or such is much easier to reason about.